ci/requests.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tests/test_requests.py b/tests/test_requests.py
```

Comment:
Hmm, is there something we can do to fix this upstream?

Reply:
Honestly I haven't investigated, but I will at least open an issue and spend a few minutes investigating